node/runtime/pangolin/src/weights/pallet_treasury_treasury.rs

```diff
@@ -0,0 +1,107 @@
+// This file is part of Darwinia.
+//
+// Copyright (C) 2018-2022 Darwinia Network
+// SPDX-License-Identifier: GPL-3.0
+//
+// Darwinia is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Darwinia is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Darwinia. If not, see <https://www.gnu.org/licenses/>.
+
+//! Autogenerated weights for `pallet_treasury`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-07-29, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("pangolin-dev"), DB CACHE: 128
+
+// Executed Command:
+// ./target/release/drml
+// benchmark
+// --header
+// .maintain/lincense-header
+// --execution
+// wasm
+// --heap-pages
+// 4096
+// --steps
+// 50
+// --repeat
+// 20
+// --chain
+// pangolin-dev
+// --output
+// node/runtime/pangolin/src/weights/
+// --extrinsic
+// *
+// --pallet
+// *
+
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weight functions for `pallet_treasury`.
```

Comment:
Can we rename `pallet_treasury_treasury.rs`, seems weird.

Reply:
This is automatically generated by the benchmark to distinguish between `KtonTreasury` and `Treasury`